scripts/release_ppa.sh

```diff
@@ -47,17 +51,34 @@ else
     branch=$1
 fi
 
+# Sanity check
+if ! grep ethereum /etc/dput.cf -q; then
+    echo "Error: Missing required entries in /etc/dput.cf (check top comment in $0 for more information)."
+    exit 1
+fi
```

Comment:
We can either keep this as a crude sanity check - or actually do it properly a few lines down :-).
If ``is_release`` you need ``[ethereum]`` and ``[ethereum-static]`` otherwise you need ``[ethereum-dev]``.

Reply:
updated :)